### ISSUE TITLE
Add instructions for adding the keyboard from the terminal

### ISSUE THREAD

**Issue by somso2e** [sinadarvi/windows-persian-keyboard-for-linux#20]
A lot of people might not use or have a GUI interface for everything on their system. So a terminal is their only choice.

Instructions:
Add the following command to your `~/.bashrc` file: 
```
setxkbmap -layout us,ir -variant ",pes_winkeys" -option grp:alt_shift_toggle
```
Run `. ~/.bashrc`.

**Comment by cs127** [comment]
Good idea, but in case #19 gets merged, change `pes_winkeys` to just `winkeys`.

**Comment by cs127** [comment]
This isn't specifically for the Windows-style Persian layout. If anyone wanted to add the "Standard" layout that already comes with X11, they would also have to do the same thing if they wanted to use the terminal.

Is this really necessary to be added to this repo?

**Comment by sinadarvi** [comment]
Implemented in f7d268e.

I kept this README-only, since the layout/install behavior did not need to change. The README now includes an "Enable From the Terminal" section with the `setxkbmap` command, explains the `pes_winkeys` variant and Alt+Shift switching option, and recommends X session startup files/autostart instead of `~/.bashrc` for persistence.

I also fixed the nearby manual installation typo from `./ir` to `./ir_patch`.
